lib/evaluate/characteristics.h

```diff
@@ -44,26 +45,18 @@ extern template class Fortran::common::Indirection<
 
 namespace Fortran::evaluate::characteristics {
 
+template<typename T> using CopyableIndirection = common::CopyableIndirection<T>;
```

Comment:
`using common::CopyableIndirection;` might work.

Reply:
You're right. I thought I had tried that but apparently not.